sandbox/src/mocks/r4/createServiceRequest/requests/createIncomplete.json

```diff
@@ -0,0 +1,87 @@
+{
+  "resourceType": "ServiceRequest",
+  "subject": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9912003888"
+    }
+  },
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/message-category-servicerequest",
+          "code": "referral"
+        }
+      ]
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "https://fhir.nhs.uk/CodeSystem/ServiceRequest-Code",
+        "code": "advice-and-guidance"
+      }
+    ]
+  },
+  "priority": "routine",
+  "intent": "order",
+  "status": "draft",
+  "extension": [
+    {
+      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-ERS-Referral",
+      "extension": [
+        {
+          "url": "sourceSystem",
+          "valueCode": "ers"
+        }
+      ]
+    },
+    {
+      "url": "https://fhir.nhs.uk/StructureDefinition/Extension-eRS-ReferralState",
+      "extension": [
+        {
+          "url": "state",
+          "valueCoding": {
+            "system": "https://fhir.nhs.uk/CodeSystem/eRS-ReferralState",
+            "code": "INCOMPLETE",
+            "display": "Incomplete"
+          }
+        }
+      ]
+    }
+  ],
+  "contained": [
+    {
+      "resourceType": "PractitionerRole",
+      "id": "requester",
+      "practitioner": {
+        "identifier": {
+          "system": "http://fhir.nhs.net/Id/sds-user-id",
+          "value": "103264992985"
+        },
+        "display": "DR AA BHATIA"
+      },
+      "organization": {
+        "identifier": {
+          "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+          "value": "C81007"
+        },
+        "display": "VERNON STREET MEDICAL CTR"
```

Comment:
We will have to make a decision on display values.

For resources we don't expose ourselves (like Organisation/Practitioner) I'm more open to this as a shortcut. But for those we do expose it would be better if they hit the API (i.e. GET HealthcareService)